src/main/java/cz/codecamp/logger/LoggerInterface.java

```diff
@@ -1,5 +1,18 @@
 package cz.codecamp.logger;
 
-public interface LoggerInterface {
+import java.time.LocalDateTime;
+
+public interface LoggerInterface extends FormatterInterface {
```

Comment:
`LoggerInterface` by právě s `FormatterInterface` neměl mít nic společného. Proto byly v poskytnutém kód oddělené :) Co loguju a jak to nakonec zobrazím jsou 2 rozdílné responsibilities.